### PR TITLE
[Doc] Release notes for v3.1.1

### PR DESCRIPTION
**Pull request by Fuzzbawls**
Update the `3.1` branch `doc/release-nodes.md` file for `v3.1.1`

### CODE REVIEW

**Comment by jakimanboy** [comment]
ACK

File: doc/release-notes.md [hunk 1]
```diff
@@ -36,377 +31,124 @@ frequently tested on them.
...
-zPIV minting now only requires 1 further mint (down from 2) to mature. zPIV mints still require 20 confirmations.  Mints also require that the 'second' mint is at least two checkpoints deep in the chain (this was already the case, but the logic was not as precise).
+### Fix spending for v1 zPIV created before block 1050020
 
+The transition to v2 zPIV and reset of the accumulators caused blocks 1050000 - 1050010 to be accumulated twice. This was causing a number v1 zPIV to not create valid witnesses, and thus were not spendable. This problem is fixed by double accumulating blocks 1050000-1050010 when creating the witness. Any user that had issues spending zPIV v1 will now be able to convert that into PIV and then zPIV v2 (if desired).
 
-### zPIV Search
+### Adjustment to staking properties to reduce orphaned blocks
 
-Users will now have the ability to search the blockchain for a specific serial # to see if a zPIV denomination has been spent or not.
+zPIV stake set to update more frequently and lowering the stake hashdrift to 30 seconds to improve number of orphans being experienced by PIVX stakers.
```
Comment (Mrs-X) [inline doc/release-notes.md]: I'd rather "reduce" the number of orphans than improve them ;-)

File: doc/release-notes.md [hunk 2]
```diff
@@ -36,377 +31,124 @@ frequently tested on them.
...
+Here is the long overdue update for PIVX to let go of OpenSSL in its consensus code. The rationale behind it is to avoid depending on an external and changing library where our consensus code is affected. This is security and consensus critical. PIVX users will experience quicker block validations and sync times as block transactions are verified under libsecp256k1.
 
+The recent [CVE-2018-0495](https://www.nccgroup.trust/us/our-research/technical-advisory-return-of-the-hidden-number-problem/) brings into question a potential vulnerability with OpenSSL (and other crypto libraries) that libsecp256k1 is not susceptible to.
 
-### Removed Growl Support
+### Write to the zerocoinDB in batches
 
-Growl hasn't been free nor needed for many years. MacOS versions since 10.8 have the OS notification center, which is still supported after this.
+Instead of using a separate write operation for each and every bit of data that needs to be flushed to disk, utilize leveldb's batch writing capability. The primary area of improvement this offers is when reindexing the zerocoinDB (`-reindexzerocoin`), which went from needing multiple hours on some system to mere minutes.
```
Comment (Mrs-X) [inline doc/release-notes.md]: ..."on some systems"... (plural)

Comment (Fuzzbawls) [reply doc/release-notes.md]: fixed

File: doc/release-notes.md (hunk 1 above)
Comment (Fuzzbawls) [reply doc/release-notes.md]: fixed

**Pull request merged**